src/vue/components/word-usage-examples/word-usage-examples-filters.vue

```diff
@@ -85,7 +85,11 @@ export default {
           .filter(wordUsageExampleItem => wordUsageExampleItem.author)
           .map(wordUsageExampleItem => wordUsageExampleItem.author)
           .filter((item, pos, self) => self.indexOf(item) == pos)
-          .slice()
+          .slice().sort((a,b) => {
+            let aT = this.calcTitle(a, 'author')
+            let bT = this.calcTitle(b, 'author')
+            return (aT < bT) ? -1 : (aT > bT) ? 1 : 0
```

Comment:
we probably ought to use localeCompare here (see https://github.com/alpheios-project/components/blob/master/src/vue/components/word-list/word-language-panel.vue#L221)